CHANGELOG

```diff
@@ -1,6 +1,10 @@
 v3.6.6 (XXXX-XX-XX)
 -------------------
 
+* Fixed slightly wrong log level for authentication
+
+* Added login event also to the standard log
```

Comment:
`* Fixed slightly wrong log level for authentication and added login event also
  to the standard log.`